Javascript/README.md

```diff
@@ -49,6 +49,23 @@ if (time < 60) {
 }
 ```
 
+There is an objective reason for this. If you write the following code
+
+```javascript
+function myFunction()
+{
+	return
+	{
+		property1: 'value1',
+		property2: 'value2'
+	};
+}
+```
+
+`return\n` will be interpreted as `return;\n` and your function will be broken. Thus, for consistency matter, we want
+any curly braces to be on the same line as whatever they open.
```

Comment:
> But you can't have block brace after a return.

That's why that rule is documented as such. I don't understand the point, sorry :(
Not trying to be a dick, but it seems we're all in agreement, yet there is a problem?

Perhaps you want the rule of curly braces on the initial line to only apply to certain cases? I think @stouf is trying to say "so let's be consistent, and do it one way only."